lib/build.js

```diff
@@ -351,7 +351,11 @@ module.exports = {
             }.bind(this), 1000);
           } else {
             var t = this.targets[this.activeTarget];
-            this.errorMatcher.set(t.errorMatch, this.replace(t.cwd), this.buildView.output.text());
+            if (t.errorFile) {
+              this.errorMatcher.set(t.errorMatch, this.replace(t.cwd), this.buildView.output.text(), this.replace(t.errorFile));
+            } else {
+              this.errorMatcher.set(t.errorMatch, this.replace(t.cwd), this.buildView.output.text());
+            }
```

Comment:
I think `error-matcher` should take care of whether `errorFile` is set or not, rather than duplicating this line